core/schema/sdk.go

```diff
@@ -92,8 +94,24 @@ func (s *moduleSchema) builtinSDK(ctx context.Context, root *core.Query, sdkName
 	case "typescript":
 		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.TypescriptSDKManifestDigestEnvName)))
 	default:
-		return nil, fmt.Errorf("%s: %w", sdkName, errUnknownBuiltinSDK)
+		sdkName, sdkVersion, hasVersion := strings.Cut(sdkName, "@")
+		if !hasVersion {
+			sdkVersion = engine.Tag
+		}
+		sdkSuffix := ""
+		if sdkVersion != "" {
+			sdkSuffix = "@" + sdkSuffix
+		}
+
+		switch sdkName {
+		case "community/elixir":
+			return s.sdkForModule(ctx, root, "github.com/dagger/dagger/sdk/elixir"+sdkSuffix, dagql.Instance[*core.ModuleSource]{})
+		case "community/php":
+			return s.sdkForModule(ctx, root, "github.com/dagger/dagger/sdk/php"+sdkSuffix, dagql.Instance[*core.ModuleSource]{})
+		}
```

Comment:
Hmmm... at this point we could even do away with the `community/` prefix, it just becomes an implementation detail that it's being fetched from git rather than bundled in the engine. The only advantage of the prefix is as a stability signal for users, but not sure if that's worth it.

Reply:
I'm not sure about this one, by prefixing `community`, we are explicit about who take care of the support, it may  happen that community SDKs implements features later than dagger's so it creates a simple way to know what's "official" and what's "community".

@shykes I think your opinion would be useful on this one

Reply:
Removed the `community` prefix.